File: DAY 8/Object.c
#include<stdio.h>
#include<string.h>
struct student{
    char name[50];
    int age;
};
void display(struct student s2){
        printf("Student Name:%s, Age:%d\n",s2.name,s2.age);
    }
int main(){
    struct student s1;
    strcpy(s1.name,"Thammi");
    s1.age=20;
    display(s1);
    return 0;
}

File: DAY 8/Object.py
class student:
    def __init__(self,n,a):
        self.name=n;
        self.age=a;
    def display(self):
        print(f"Student Name:{self.name} , Age: {self.age}")
s1=student("Thammi",20) 
s1.display()
